Classes/ContextMenu/PasteReferenceItemProvider.php

```diff
@@ -111,8 +103,14 @@ protected function getAdditionalAttributes(string $itemName): array
      */
     public function addItems(array $items): array
     {
-        $this->initialize();
+        $this->itemsConfiguration['pasteReference'] = [
```

Comment:
Combined with the removal of the hard rest above, this now add a new item to a whole list of items configuration stemming from the parant.

The side effect is, that in case other providers have removed items that would now be added again.

Thus, this rework comes with side effects which cannot be intentional.

Reply:
the change is reverted and includes only the local definition again in 188625fc